samza-core/src/main/java/org/apache/samza/operators/impl/OperatorImplGraph.java

```diff
@@ -18,6 +18,7 @@
  */
 package org.apache.samza.operators.impl;
 
+import com.google.common.collect.Lists;
```

Comment:
`Lists.reverse` is nicer as it does not mutate the passed in list. We appear to be having a guava dependency anyways in samza-core